include/NovelRT/LoggingService.h

```diff
@@ -4,8 +4,17 @@
 #ifndef NOVELRT_LOGGINGSERVICE_H
 #define NOVELRT_LOGGINGSERVICE_H
 
-#ifndef NOVELRT_H
-#error Please do not include this directly. Use the centralised header (NovelRT.h) instead!
+#if defined(_MSC_VER)
+#pragma warning(push)
+#pragma warning(disable : 4275)
+#endif
```

Comment:
Why was this added?

Reply:
Because I was getting warnings from the dependency so I suppressed them iirc.